plugin/driver/driver.go

```diff
@@ -125,20 +126,31 @@ func (driver *driver) JoinEndpoint(j *api.JoinRequest) (response *api.JoinRespon
 		return
 	}
 
-	var bridge *netlink.Bridge
 	if maybeBridge, err := netlink.LinkByName(WeaveBridge); err != nil {
-		err = fmt.Errorf(`bridge "%s" not present`, WeaveBridge)
+		error = fmt.Errorf(`bridge "%s" not present`, WeaveBridge)
```

Comment:
Can we get rid of the `error` named return argument? Every time we have to return an error we're using two lines instead of one, and the `err/error` confusion means there were *two* bugs in the original code that you've had to fix!